build.gradle

```diff
@@ -105,7 +105,7 @@ allprojects {
 		maven { url 'https://repo.spring.io/libs-milestone' }
 		if (version.endsWith('-SNAPSHOT')) {
 			maven { url 'https://repo.spring.io/libs-snapshot' }
-			maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
+			// maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
```

Comment:
May be there is just enough to remove this repo altogether?

Reply:
Problem is remembering where it is the next time I have to use a snap.